packages/react-avatar/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './Avatar';
+export * from './utils';
```

Comment:
You're getting a warning to make this refer to a physical file, so this should probably be:

```suggestion
export * from './utils/index';
```